templates/cluster-template-powervs.yaml

```diff
@@ -202,6 +206,10 @@ spec:
         name: "${IBMPOWERVS_IMAGE_NAME}"
       network:
         name: "${IBMPOWERVS_NETWORK_NAME}"
+      memory: ${IBMPOWERVS_MEMORY:="4"}
+      processors: ${IBMPOWERVS_PROCESSORS:="0.25"}
+      sysType: ${IBMPOWERVS_SYSTYPE:="s922"}
+      procType: ${IBMPOWERVS_PROCTYPE:="shared"}
```

Comment:
can these defaults be part of another PR?

Reply:
Done